tools/spelling_dictionary.txt

```diff
@@ -625,9 +626,12 @@ istio
 istream
 istringstream
 iteratively
+ja
```

Comment:
FWIW it would be nice to somehow exempt these lines from the spell checker. @zuercher any thoughts here?

Reply:
OK we should add this later.

Reply:
Sorry, digging myself out of the vacation messaging deluge. We could add some magic comments to exclude a region of a file for cases like this. I'll file an issue.